src/index.html

```diff
@@ -11,7 +11,158 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="block">
+        <legend>Personal information</legend>
+        <div class="input-block">
+          <label for="surname">Surname:</label>
+          <input
+            type="text"
+            id="surname"
+            name="surname"
+            autocomplete="off"
+          >
+        </div>
+        <div class="input-block">
+          <label for="name">Name:</label>
+          <input
+            type="text"
+            id="name"
+            name="name"
+            autocomplete="off"
+          >
+        </div>
+        <div class="input-block">
+          <label for="age">How old are You?</label>
+          <input
+            type="number"
+            id="age"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </div>
+        <div class="input-block">
+          <label for="birth">Full date of birth:</label>
+          <input
+            type="date"
+            id="birth"
+            name="birth"
+          >
+        </div>
+        <div class="input-block">
+          <label for="term">I accept the term of the agreement</label>
+          <input
+            type="checkbox"
+            name="term"
+            id="term"
+          >
+        </div>
+
+        </label>
+      </fieldset>
+      <fieldset class="block">
+        <legend>Registration</legend>
+        <div class="input-block">
+          <label for="email">E-mail:</label>
+          <input
+            type="email"
+            id="email"
+            name="email"
+            placeholder="email@example.com"
+            autocomplete="off"
+            required
+          >
+        </div>
+        <div class="input-block">
+          <label for="password">Password:</label>
+          <input
+            type="password"
+            id="password"
+            name="password"
+            minlength="5"
+            maxlength="10"
+            autocomplete="off"
+          >
+        </div>
+
+      </fieldset>
+      <fieldset class="block">
```

Comment:
```suggestion
      </fieldset>

      <fieldset class="block">
```

Add empty line between multiple sibling blocks